helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableReplica.java

```diff
@@ -151,22 +152,24 @@ public static String generateReplicaKey(String resourceName, String partitionNam
 
     Map<String, Integer> partitionCapacity = capacityMap.get(partitionName);
     if (partitionCapacity == null) {
-      partitionCapacity = capacityMap.get(ResourceConfig.DEFAULT_PARTITION_KEY);
+      partitionCapacity =
```

Comment:
Let's clearly document the behavior here as a comment - I think what we used to have as a warn log could be changed to an inline comment here. Also, let's say that if partition capacity usage is not found in ResourceConfig, we will try to get it from ClusterConfig.

Reply:
Sure. Sounds good.